data/human/news.txt

```diff
@@ -1990,6 +2113,83 @@ news "conspiracy theorist"
 
 
 
+news "player's crew member"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	to show
+		"flagship crew" > 40
+	name
+		word
+			"Crew member"
+	message
+		word
+			`"`
+		word
+			"I'm serving on board the '<ship>,' but I've never met the captain. Can you imagine that?"
+			"Hey captain, good to see you here. I'm looking forward to the next trip, and, of course, my 100 credits a day."
+			"You look like a poor excuse for a captain. I'm serving onboard the '<ship>' and I'm sure that my captain is far better dressed than you."
+		word
+			`"`
+
+
+
+news "sticker on ship"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	name
+		word
+			"Sticker on ship"
+	message
+		word
+			"I brake for asteroid snails."
+			"Student pilot."
+			"Babe onboard."
+			"Baby onboard."
+			"There's no place like [insert home planet here]."
+			"This is not a bumper sticker, this is a rust retardant patch."
+			"Air leak seal, do not remove."
+
+
+
+news "label on ship"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	name
+		word
+			"Label on ship"
+			"Warning label on ship"
+	message
+		word
+			"Angles morts - Blind spots from cockpit."
+			"Stay clear of ramscoop intake."
+			"Stay out of thruster exhaust stream."
+			"Stay away from thruster outlets during takeoff."
+			"Door opens outwards."
+			"Remove red ribbons before flight."
+			"Nuclear reactor compartment: NEA certification is required for servicing this part of the ship."
+			"Compartiment du reacteur nucleaire: certification AEN requise pour l'entretien de cette partie du navire."
+
+
+
+news "logo on ship hull"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	name
+		word
+			"Logo on the hull of a ship"
+			"Print on a spaceship"
+	message
+		word
+			"Justins Time - Just in time delivery"
+			"Francis Fries - Food spacetrucker"
+			"Bobby Bughunter - Extermination services"
+			"Luna Ticks – Pest control"
+			"Orion's Belts – The fashion of the future"
+			"Comet Cleaners – Brightest laundry in the galaxy" 
+			"Cosmo Joe's – Cosmic Coffee out of this world"
```

Comment:
```suggestion
			"Luna Ticks - Pest control"
			"Orion's Belts - The fashion of the future"
			"Comet Cleaners - Brightest laundry in the galaxy"
			"Cosmo Joe's - Cosmic Coffee out of this world"
```

Reply:
Whoops, long dashes were my bad. Huh, I'd not considered that connotation 🤔.